crates/matrix-sdk-ui/src/timeline/traits.rs

```diff
@@ -76,7 +76,7 @@ pub(super) trait RoomDataProvider:
     fn own_user_id(&self) -> &UserId;
     fn room_version(&self) -> RoomVersionId;
 
-    fn crypto_context_info(&self) -> BoxFuture<'_, CryptoContextInfo>;
+    fn crypto_context_info(&self) -> BoxFuture<'_, Option<CryptoContextInfo>>;
```

Comment:
If you're going to keep this, please can you document what `None` and `Some` mean?

Reply:
Not an Option anymore https://github.com/matrix-org/matrix-rust-sdk/pull/4275/commits/85b907e6a415a5206c466dc012ed93dde72b2464